### DISCUSSION TITLE
How do I include a file in the root directory of my blog?

### DISCUSSION THREAD

**Discussion by sandsoncosta** [hbstack/.github#93] [Q&A] [answered]
I need to include ads.txt in the root of my blog, but I get the error below:
![image](https://github.com/user-attachments/assets/0e39f0af-f8ca-46dd-bff0-9f2ea7f63333)

My github:
https://github.com/sandsoncosta/sandsoncosta.github.io
![image](https://github.com/user-attachments/assets/b15d34a2-0aa4-4fca-87e7-57bf271a55ac)

**Comment by razonyang** [comment, marked as answer]
Move `ads.txt` to `static/ads.txt` should work, the `static` folder is public to web that URLs relative to the site baseURL.

**Comment by sandsoncosta** [comment]
Thanks for the help as always. It worked.
